main.py

```diff
@@ -35,8 +35,18 @@ async def new_message_listener(client, event):
 
     matched_keywords = find_intersections(lemmas)
 
-    post_message_to_db_archive(event.text, (' ').join(lemmas), f"https://t.me/{event.chat.username}/{event.id}",
-                               bool(matched_keywords), EnvProcessor.message_collecting_is_on())
+    try:

```

Comment:
for the future i suggest moving try-catch block inside the catebi_adapter.post_message_to_db_archive method
so, this will also require moving the logging (debug method) to a separate service